src/components/data-management/AddMetadataButton.jsx

```diff
@@ -17,31 +21,52 @@ const AddMetadataButton = ({ samplesTableRef }) => {
   const isSubsetted = activeExperiment?.isSubsetted;
   const samples = useSelector((state) => state.samples);
   const selectedTech = samples[activeExperiment?.sampleIds[0]]?.type;
-  const [uploadModalVisible, setUploadModalVisible] = useState(false);
 
-  const uploadFiles = (file) => {
+  const [uploadModalVisible, setMetadataUploadModalVisible] = useState(false);
+  const [cellLevelUploadVisible, setCellLevelUploadVisible] = useState(false);
+
+  const onUploadMetadataTrack = (file) => {
     dispatch(uploadMetadataFile(activeExperimentId, file));
-    setUploadModalVisible(false);
+    setMetadataUploadModalVisible(false);
+  };
+
+  const onUploadCellLevelMetadata = async (file) => {
+    const body = {
+      name: file.name,
+    };
+
+    const signedUploadURL = await dispatch(createCellLevelMetadata(activeExperimentId, body));
+    await putPartInS3(file, signedUploadURL, (progressEvent) => console.log('uploaded ', progressEvent.loaded, ' of total ', progressEvent.total));
```

Comment:
We don't need to handle the progress events in any way, so feel free to not pass the last param

Reply:
Oh I see `onUploadProgress` doesn't have a default value, you could set its default to `null` if you see that axios doesn't complain or set it to something like `() => {}` or whatever is the best way to pass identity if axios does want a function

Reply:
I wanted to keep it like this for now until we add the modal with the information and decide if we want the progress or not there. Lets keep it like this until the next PR which finalises this functionality?

Reply:
ok, then could you leave a comment saying it is a placeholder?